### PR TITLE
New package: lemonbuddy-1.4.4_1

### PR DESCRIPTION
**Pull request by jaagr**

### CODE REVIEW

File: srcpkgs/lemonbuddy/template [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Template file for 'lemonbuddy'
+pkgname=lemonbuddy
+version=1.1.2
+revision=1
+_i3ipcpp_version=0.1.3
+_lemonbar_version=1.1
+wrksrc="${pkgname}-${version}"
+# gcc: posix_memalign declaration is incompatible with musl for c++
+only_for_archs="x86_64 i686 armv7l armv6l"
```
Comment (Vaelatern) [inline srcpkgs/lemonbuddy/template]: Please change the above line to a `broken` statement for musl.
```
case "${XBPS_TARGET_MACHINE}" in
	*-musl) broken="posix_memalign declaration is incompatible with musl for c++";;
esac
```

But a link to an example failing build would be better.

File: srcpkgs/lemonbuddy/template [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Template file for 'lemonbuddy'
+pkgname=lemonbuddy
+version=1.1.2
+revision=1
+_i3ipcpp_version=0.1.3
+_lemonbar_version=1.1
+wrksrc="${pkgname}-${version}"
```
Comment (Vaelatern) [inline srcpkgs/lemonbuddy/template]: Remove

File: srcpkgs/lemonbuddy/template (hunk 1 above)
Comment (jaagr) [inline srcpkgs/lemonbuddy/template]: Ah, didn't know about the broken variable, will fix that..

"...an example failining build..."?

Comment (jaagr) [inline srcpkgs/lemonbuddy/template]: nevermind, found some other templates doing the same thing

File: srcpkgs/lemonbuddy/template [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+homepage="https://github.com/jaagr/lemonbuddy"
+distfiles="
+	https://github.com/jaagr/lemonbuddy/archive/${version}.tar.gz
+	https://github.com/jaagr/i3ipcpp/archive/v${_i3ipcpp_version}.tar.gz
+	https://github.com/jaagr/bar/archive/v${_lemonbar_version}.tar.gz"
+checksum="
+	da295c5361d6b29a6bc89b2b16951fa6a48f1396057acf85c42239c8602c9676
+	fb5ea31bf2d008065512f12ab9d9cc8304b2a4e6ba2ae93082df29712f372449
+	d9073ecc3d2638804a723d47c74666ba3cd1c3771dfe254809de01ec5c428d4a"
+nopie=yes
```
Comment (Vaelatern) [inline srcpkgs/lemonbuddy/template]: Is it not possible to make this obey our CFLAGS and LDFLAGS?

Comment (jaagr) [inline srcpkgs/lemonbuddy/template]: Yeah, I'll fix that

File: srcpkgs/lemonbuddy/template [hunk 4]
```diff
@@ -0,0 +1,38 @@
+# Template file for 'lemonbuddy'
+pkgname=lemonbuddy
+version=1.2.0
+revision=1
+_i3ipcpp_version=0.1.3
+_lemonbar_version=1.1
+build_style=cmake
+configure_args="-DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++"
```
Comment (xtraeme) [inline srcpkgs/lemonbuddy/template]: This line is unnecessary.

File: srcpkgs/lemonbuddy/template [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+distfiles="
+	https://github.com/jaagr/lemonbuddy/archive/${version}.tar.gz
+	https://github.com/jaagr/i3ipcpp/archive/v${_i3ipcpp_version}.tar.gz
+	https://github.com/jaagr/bar/archive/v${_lemonbar_version}.tar.gz"
+checksum="
+	7260a0ebc3d7325c5fee44a1f10f397ac350d9d2ed9ce5677bce61f844a3a714
+	fb5ea31bf2d008065512f12ab9d9cc8304b2a4e6ba2ae93082df29712f372449
+	d9073ecc3d2638804a723d47c74666ba3cd1c3771dfe254809de01ec5c428d4a"
+
+build_options="alsa_support i3_support mpd_support network_support"
```
Comment (xtraeme) [inline srcpkgs/lemonbuddy/template]: Those build options are named consistently, rename them as:

`alsa, i3, mpd, network`

File: srcpkgs/lemonbuddy/template [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+distfiles="
+	https://github.com/jaagr/lemonbuddy/archive/${version}.tar.gz
+	https://github.com/jaagr/i3ipcpp/archive/v${_i3ipcpp_version}.tar.gz
+	https://github.com/jaagr/bar/archive/v${_lemonbar_version}.tar.gz"
+checksum="
+	dfbeb74aaac49cb0f3c5c98466aadf2477c6a709025174e11e1da7ad7edafef4
+	fb5ea31bf2d008065512f12ab9d9cc8304b2a4e6ba2ae93082df29712f372449
+	d9073ecc3d2638804a723d47c74666ba3cd1c3771dfe254809de01ec5c428d4a"
+
+build_options="alsa i3 mpd network"
```
Comment (xtraeme) [inline srcpkgs/lemonbuddy/template]: https://travis-ci.org/voidlinux/void-packages/jobs/138930962#L735

i3 support is disabled.

**Pull request merged**